src/EnergyPlus/CMakeLists.txt

```diff
@@ -711,7 +728,10 @@ add_custom_target(
         -D "EnergyPlus_embeddable_epJSON_schema:PATH=$<TARGET_FILE:generate_embeddable_epJSON_schema>"
         -P "${PROJECT_SOURCE_DIR}/scripts/dev/generate_embeddable_epJSON_schema/generate_embedded_epJSON_schema.cmake"
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
-  DEPENDS generate_embeddable_epJSON_schema)
+  DEPENDS generate_embeddable_epJSON_schema epjson_schema "${CMAKE_CURRENT_SOURCE_DIR}/InputProcessing/EmbeddedEpJSONSchema.in.cc")
```

Comment:
Similar comment to the other `add_custom_command`. This should include `BYPRODUCTS`, `VERBATIM`, and maybe `COMMENT`.
See [this code as an example](https://github.com/NREL/EnergyPlus/commit/03c3e02602ba47a9f5bf056caf5cd3f88fda78d1#diff-55856ea372630af1ffb24adf007a830b73b57f27bfbd5783767e425df75c59aeR28-R31).